### PR TITLE
BiDi node serialization fix

### PR DESCRIPTION
**Pull request by sadym-chromium**
Use `recursive_compare` to tolerate additional fields (`sharedId`)

### CODE REVIEW

File: webdriver/tests/bidi/script/call_function/result_node.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import pytest
 from webdriver.bidi.modules.script import ContextTarget
+from ... import recursive_compare1
```
Comment (foolip) [inline webdriver/tests/bidi/script/call_function/result_node.py:3]: I guess this didn't work?

```suggestion
from ... import recursive_compare
```

**Review by foolip — APPROVED** [review]
LGTM if it works :)

**Review by whimboo — APPROVED** [review]
@sadym-chromium is that urgent to have for you? I'm having the exact same changes in my `sharedId` patch on https://phabricator.services.mozilla.com/D169492. I'm requesting review soon so hoping to have it by end of the week. But if you need it we could land this change ahead for sure.

**Comment by whimboo** [comment]
Oh actually `evaluate` would need the exact same changes as well.

**Comment by sadym-chromium** [comment]
> Oh actually `evaluate` would need the exact same changes as well.

It's not urgent, and we will gladly wait for your roll

**Pull request closed without merging**